build.sbt

```diff
@@ -6,14 +6,25 @@ def commonSettings = Seq(
   isSnapshot := true,
   scalaVersion := "2.11.6",
   resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
-  libraryDependencies += "org.specs2" %% "specs2-core" % "2.4.14" % "test"
+  libraryDependencies += "org.scalatest" %%% "scalatest" % "3.0.0-M9" % "test"
 )
 
+lazy val root =
```

Comment:
There's actually a problem here.  It's finding the 'test' code in this project and trying to compile it, but failing.   We should either move all the code into some kind of sub-directory or DISABLE compilation/source on this project.